person.rb

```diff
@@ -13,6 +13,10 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     super()
   end
 
```

Comment:
Kindly ensure that a person has many rentals. This will require adding an instance variable as thus `@rentals = []`.

Also, add the rentals variable to the attribute accessor thus `attr_accessor :name, :age, :rentals`